modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/rest-openapi.yaml

```diff
@@ -1973,6 +1973,79 @@ info:
     version: v1.0
 openapi: 3.0.0
 paths:
+    "/attachment/by-externalReferenceCode/{externalReferenceCode}":
+        delete:
+            operationId: deleteAttachmentByExternalReferenceCode
+            parameters:
+                - in: path
+                  explode: false
+                  name: externalReferenceCode
+                  required: true
+                  schema:
+                      description:
+                          The resource external reference code
+                      example: AB-34098-789-N
+                      minLength: 1
+                      type: string
+                  style: simple
+            responses:
+                204:
+                    description:
```

Comment:
@dsitu actually the pattern for the delete method is this https://github.com/liferay/liferay-portal/blob/6537b29aa21c90775cea4537338a34842971e14d/modules/apps/headless/headless-admin-content/headless-admin-content-impl/rest-openapi.yaml#L444C1-L449C40 . 
Also please move the yaml file changes to a separate commit and the commit message finishes with _(yaml file)_